src/index.html

```diff
@@ -17,6 +17,7 @@ <h1>Sokoban</h1>
     </div>
     <div id="game" class="hide">
       <div id="controls">
+        <button id="rewind">rewind (<kbd>z</kbd>)</button>
```

Comment:
Rewind and undo both say they're mapped to the Z key. I suggest we drop undo and use rewind behavior throughout (but call it undo).

This still uses the old `index.html`. #49 makes a pretty radical refactor that moves into a fully JS-driven single page app (SPA). So I'd keep all of the front-end code the same and simply have the glue code call `rewind` for now. If we want to add `undo` as a separate button, we can do that later, assuming we keep both in the engine (see other comments below).